backend/main.py

```
@@ -163,7 +164,21 @@ def get_recent_audit_logs(self, limit): return []
     RL_ENV_READY = True
     logger.info("✅ AI FIRM & RL CORE OPERATIONAL")
 except Exception as e:
+
     logger.error(f"❌ AI Firm core initialization failed: {e}")
+    # Fallback for tests
+    class MockDebateEngine:
+        async def conduct_debate(self, symbol, context):
```

Ruff (0.15.2)

[warning] Unused method argument: `context`

(ARG002)